.circleci/config.yml

```diff
@@ -1,52 +1,21 @@
 version: 2
 jobs:
-  build:
+  lint:
     docker:
-      # specify the version you desire here
-       - image: circleci/python:3.6
-
-    working_directory: ~/repo
-
+       - image: circleci/python:3
     steps:
       - checkout
-
-      # Download and cache dependencies
-      - restore_cache:
-          keys:
-          - v1-dependencies-{{ checksum "requirements.txt" }}
-          # fallback to using the latest cache if no exact match is found
-          - v1-dependencies-
-
       - run:
-          name: Install dependencies
-          command: |
-            sudo apt-get install -y python-pip virtualenv
-            virtualenv venv
-            . venv/bin/activate
-            pip install --upgrade setuptools wheel
-
-
+          name: Install requirements
+          command: sudo pip install -r requirements.txt
       - run:
-          name: Create package
-          command: python setup.py sdist bdist_wheel
+          name: Run pylint
+          command: pylint src
 
-      - save_cache:
-          paths:
-            - ./venv
-          key: v1-dependencies-{{ checksum "requirements.txt" }}
 
-      - run:
-          name: Push package
-          command: |
-            curl -F package=@$(ls -t dist/*.whl) https://$GEMFURY_TOKEN@push.fury.io/silvercar/
 
 workflows:
   version: 2
-  build-publish:
+  build:
```

Comment:
who no more publish?

Reply:
I figured since this is a public repo, it doesn't make sense to gate the package behind gemfury. What do you think?

Reply:
yeah, that's what I figured, but wanted to check if that was your reasoning. Ultimately we should to publish somewhere though